var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -65,6 +65,14 @@ class Rust(Package):
     depends_on('libssh2')
     depends_on('libgit2')
 
+    # Pre-release Versions
+    version('master', branch='master', submodules=True)
+
+    # These version strings are officially supported, but aren't explicitly
+    # listed because there's no stable checksum for them.
+    # version('nightly')
```

Comment:
It would be nice if I could get these versions to show up in `spack info rust`. Is there some way to specify that a version cannot be checksummed?

Reply:
Hmm, you could just add it with a fake checksum like you did with resources. @tldahlgren @tgamblin may have ideas for how to handle this. We have a similar problem with `texlive` which only provides a single download tarball that's a moving target.